### ISSUE TITLE
Add new tab for checklist

### ISSUE THREAD

**Issue by skacco** [nextcloud/deck#1953]
Writing a checklist is very useful (with this syntax in the description "- [ ]") to manager task list but it is a problem to use because an user need to have modify permission to allow to check the box.
It is not correct because the user could also delete or modify single task.
It will be better if we could have a dedicated tab only for check list

**Comment by stefan-niedermann** [comment]
Then one would always have to jump around and the checkboxes wouldn't be visible directly but only after an additional click 😕 .

Also: wirte permission is write permission.Manipulating check items can be even more dangerous than deleting a card because it is more subtile and people wouldn't see it immediately. I think this is not a question of permissions but only UI / UX therefore.

**Comment by juliushaertl** [comment]
duplicate of #1937 

I also agree with @stefan-niedermann that toggling checkboxes and editing todo entries should definitely be part of the write permission.
